### ISSUE TITLE
Add support for rails 4

### ISSUE THREAD

**Issue by scottish613** [ankane/multiverse#2]
Exciting to see this new gem!

Unfortunately our project is still on rails `4.1.16`. Have you explored how easily/well `multiverse` could support older versions of rails?

**Comment by ankane** [comment]
Hey @scottish613, I created a `rails42` branch with tests passing for Rails 4.2.10. It may work with Rails 4.1 (after updating gemspec) but haven't tested it. Since official Rails support for 4.x series ended (except for severe security issues), I don't plan to maintain support for any 4.x versions.

**Comment by scottish613** [comment]
Thanks for your help, @ankane!

Seems to work with 4.1 as well. I forked to update the gemspec, and made some additional changes to get `db:migrate:status` working as well (not sure if that was working with Rails 5.x). I'll keep the fork separate since you're not planning to support 4.x. Feel free to do what you'd like with the `rails42` branch.

Closing - in response to your statement about your plans for 4.x support.
